.gitattributes

```diff
@@ -1 +1 @@
-*.pbxproj binary merge=union
+*.pbxproj diff merge
```

Comment:
Treating the Xcode project file as a binary is an outdated process. It's better to let git treat it as plaintext and apply some special diffing attributes to help minimize conflicts.